config/metadata/rulesMeta.yaml

```diff
@@ -609,7 +609,11 @@ groups:
         description: >
           The field to check. This can name any field in the trace. If the field
           is not present, then the condition will not match. The comparison is
-          case-sensitive.
+          case-sensitive. The field can also include span context such as root 
+          span context (e.g. `root.http.status`). Specifying root span context 
+          limits matching conditions to those that match the root span. Using root
+          span context also means that if the trace matches, every span within the
+          trace will match since they share a root span.
```

Comment:
```suggestion
          case-sensitive. 
          
          The field can also include a prefix that changes the span used for evaluation of the field.
          The only prefix currently supported is `root`, as in `root.http.status`. Specifying `root.` 
          causes the condition to be evaluated against the root span. 
          
          For example:
          
          Field: root.url
          Operator: contains
          Value: login

          This condition will evaluate to true when the field called `url` on the root span of the 
          trace contains `login`.
          
          The rule setting `Scope: span` does not change the meaning of this prefix --
          the condition is still evaluated on the root span and is treated as if it were part of the  
          span being processed. 
```

Reply:
I'm not finding the terms "limit" and "span context" to be the right phrasing for this. I tried something above -- feel free to wordsmith it if it's not working for you.

I also think we should tweak one of the examples in `rules_complete.md`.

Reply:
I played with this a little bit, but I think your wording is good here. I'll go ahead and commit this change.